test/test-ocp.bats

```diff
@@ -37,6 +37,12 @@ teardown_file() {
   fi
 }
 
+@test "olm: es-indexing=true" {
+  run_cmd ${KUBE_BURNER_OCP} olm --log-level debug --uuid=${UUID} --iterations 20 --catalogImage registry.redhat.io/redhat/redhat-operator-index:v4.18 ${COMMON_FLAGS}
+  check_metric_value podLatencyMeasurement nodeLatencyMeasurement
+  # run_cmd oc delete clustercatalog --all
```

Comment:
Please, don't run this command, as it can affect other operators that are currently installed in the CI cluster